boards/arm/stm32f746g_disco/Kconfig.defconfig

```diff
@@ -15,6 +15,9 @@ config NET_L2_ETHERNET
 
 endif # NETWORKING
 
+config KSCAN
+	default y if DISPLAY
+
 config I2C
 	default y if KSCAN
```

Comment:
This one could be put under `if KSCAN` while you're at.

Reply:
Grouped